tests/test_neuroprobe_popt_pipelines/test_neuroprobe_clean_laplacian.py

```
@@ -0,0 +1,133 @@
+import json
+import os
+from types import SimpleNamespace
+
+import torch
+
+os.environ["ROOT_DIR_BRAINTREEBANK"] = "/tmp"
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)